packages/manager/modules/hub/src/components/tile/tile.less

```
@@ -0,0 +1,79 @@
+.ovh-manager-hub-tile {
```

Could you scope it to the component ? 

```suggestion
ovh-manager-hub-tile {
```

---

Sorry for misunderstanding your previous. This is done now :)